.github/workflows/nightly-release.yml

```diff
@@ -0,0 +1,173 @@
+name: Nightly Release
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  "0 0 * * 1-5"
```

Comment:
I wonder if we should launch the job after the one of dynawo ?

Reply:
we can do it in another pr

Reply:
You are right, maybe we should delay the nightly releases of dynawo dependant projects 3 or 4 hours, to ensure we always try to build using _today's_ dynawo assets, instead of _yesterday's_.

Reply:
could you do the pr ? thanks

Reply:
maybe we can launch dynawo at 10pm also instead of midnight